docs/source/operations/walkthrough/10-prerequisites.rst

```diff
@@ -0,0 +1,120 @@
+*************
+Prerequisites
+*************
+
+Substra version
+===============
+
+Substra is a set of microservices which are together issued a version number; but, since we are installing the services one by one, we need to know the actual version of each one.
+
+Check the :ref:`compatibility table` for the Helm chart version needed for the orchestrator, backend and frontend. The corresponding Docker app version is already configured in there, so it's all you need.
+
+Local tools
+===========
+
+You'll need:
+ - kubectl & Helm installed
+ - to add the Substra helm repository:
+   
+   .. code-block:: shell
+   
+      helm repo add substra https://substra.github.io/charts/
+      helm repo update
+
+.. Leaving kubectl and helm purposefully unlinked because if you need a link you're definitively not ready for the rest of this
+
+And also:
+ - ``curl`` or similar for making sure the HTTP endpoints work 
+ - `gRPCurl <https://github.com/fullstorydev/grpcurl>`_ for making sure the gRPC endpoint works
+
+
+Infrastructure
+==============
+
+Substra is a federated learning tool and as such it makes little sense to have only one node running, or nodes running on the same cluster merely separated by a namespace.
+
+Therefore our production deployment will run on two separate Kubernetes clusters. They will of course need to be connected somehow -- we will use the internet.
+
+You will need to be able to give hostnames to endpoints. On the internet, this means owning a domain name and setting up DNS -- **everytime you see** ``DOMAIN``, **it means your own domain** you are setting this up under.
```

Comment:
I hadn't thought about it much, it makes sense that simpler constructions are better for clarity.

However rereading myself I use the future tense either for things that will be in effect throughout the document, or things that will need to be carried out later -- there might be value in distinguishing these from straight actions that are in the present tense.

Do you have some links about this?

Reply:
Of course I cannot find again our internal guidelines (maybe @RomainGoussault you know where those are?). 

Here are [Google's guidelines](https://developers.google.com/style/tense) for exemple, and I think there were the ones linked in our guidelines (but I can't check since I cannot find those :( )